### ISSUE TITLE
Dreamview can't work with apollo8.0 update

### ISSUE THREAD

**Issue by Michael-Fuu** [guardstrikelab/carla_apollo_bridge#136]
## Describe the bug

Dreamview can't work with apollo8.0 update

## Steps to Reproduce the Problem

1. Run this command in the container to start Dreamview: `./scripts/bootstrap.sh` 
2. Run this command in the container : `source cyber/setup.sh` 
3. Start the bridge and spawn the ego vehicle: `python main.py`
4. Start a co-simulation: `Open apollo client: http://localhost:8888`
<img width="1435" alt="image" src="https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/b752f38a-a943-465b-9f4a-5f4054c14d30">
no bugs with main.py, however dreamview can't obtain car initial position and map data.

![image](https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/6d30c61f-b2a8-4d71-84f2-d309b3a40c3f)
below is cybermonitor output:
<img width="489" alt="image" src="https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/5718fcd2-6722-4e75-bd22-21f278b0bc74">

## Environments (please complete the following information):
 - System info: [use `uname --all` on LInux]
 - Cuda version [use `nvidia-smi`]
 - Docker version [use `docker -v`]
 - Docker-compose version [use `docker-compose -v`]
 - Versions of other dependent software...

## What version of carla_apollo_bridge?

> 0.2.x
> 0.1.x
> edge: output of `git describe --dirty`

## Release Note

RELEASE NOTE:

**Comment by SEEKERXC** [comment]
Make sure you started the following modules:
Routing, Localization, Planning, Control
I guess that you didn't start Localization module.
If this issue still happens, switch to another map and switch back, and then open the modules.

**Comment by Michael-Fuu** [comment]
> Make sure you started the following modules: Routing, Localization, Planning, Control I guess that you didn't start Localization module. If this issue still happens, switch to another map and switch back, and then open the modules.

I tried and the localization module indeed can't work.
In mainboard_default.INFO:
<img width="661" alt="image" src="https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/634210b3-7fa5-4efc-b40c-8f5a4b562aac">
I search the problem and find https://github.com/ApolloAuto/apollo/issues/2426, in modules/localization/conf//localization_config.pb.txt, change MSF to RTK.
and In apollo/modules/dreamview/conf/hmi_modes/mkz_standard_debug.pb.txt change MSF to RTK.
the localization module can work normally?
![image](https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/52ea04ca-c22d-4e3a-952a-6287658d1e64)
however, after python main.py, the dreamview still can't obtain location and map data.
when I check cybermonitor:
<img width="625" alt="image" src="https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/3a58d29a-ddcd-4177-bb47-30c00575ca82">
I find something wrong with this channel:
<img width="525" alt="image" src="https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/fd572f0e-898a-42fd-99c6-9123f462ef83">

**Comment by SEEKERXC** [comment]
I just tested again, Localization status Error doesn't have real effect. But we will fix it soon.
Can you restart apollo or dreamview and try again?

**Comment by SEEKERXC** [comment]
I just tested again, it does have this problem. It can be solved like this: switch the map to carla_town04, and then switch back to it

**Comment by Michael-Fuu** [comment]
> 

yes, it works! but when you flash the webpage it lost location and map data again.
what's more, the camera have bugs.
![image](https://github.com/guardstrikelab/carla_apollo_bridge/assets/55577878/76572db0-efd3-48fa-9a04-9bfa3bf253ee)

**Comment by SEEKERXC** [comment]
Does "select channel" help?

**Comment by Michael-Fuu** [comment]
> Does "select channel" help?

no, the reason seems something changes of camera config from apollo7 to apollo8.

**Comment by SEEKERXC** [comment]
> > Does "select channel" help?
> 
> no, the reason seems something changes of camera config from apollo7 to apollo8.

OK. Thank you very much for your feedback, we hurry up to fix this issue.

**Comment by LCaptainJackSparrow** [comment]
> > 
> 
> yes, it works! but when you flash the webpage it lost location and map data again. what's more, the camera have bugs. ![image](https://user-images.githubusercontent.com/55577878/263204348-76572db0-efd3-48fa-9a04-9bfa3bf253ee.png)

Can you provide detailed steps? 
I  changed  MSF to RTK In apollo/modules/dreamview/conf/hmi_modes/mkz_standard_debug.pb.txt and I also changed town to  Town04 in  apollo/modules/carla_bridge/settings.yaml and I restart apollo or dreamview. But the dreamview still can't obtain location and map data.

**Comment by Michael-Fuu** [comment]
> > > 
> > 
> > 
> > yes, it works! but when you flash the webpage it lost location and map data again. what's more, the camera have bugs. ![image](https://user-images.githubusercontent.com/55577878/263204348-76572db0-efd3-48fa-9a04-9bfa3bf253ee.png)
> 
> Can you provide detailed steps? I changed MSF to RTK In apollo/modules/dreamview/conf/hmi_modes/mkz_standard_debug.pb.txt and I also changed town to Town04 in apollo/modules/carla_bridge/settings.yaml and I restart apollo and dreamview. But the dreamview still can't obtain location and map data.

you should change map 04 in dreamview rather than in settings.yaml.

**Comment by Michael-Fuu** [comment]
> > > Does "select channel" help?
> > 
> > 
> > no, the reason seems something changes of camera config from apollo7 to apollo8.
> 
> OK. Thank you very much for your feedback, we hurry up to fix this issue.

Hi, I have tried a long time to solve the problem of "camera view missing in apollo8 dreamview" and can't find a solution, do you also find the problem or your environment is ok with that? Any suggestion may be helpful for me, thanks
